hack/cherry_pick_pull.sh

```diff
@@ -28,12 +28,13 @@ cd "${KUBE_ROOT}"
 declare -r STARTINGBRANCH=$(git symbolic-ref --short HEAD)
 declare -r REBASEMAGIC="${KUBE_ROOT}/.git/rebase-apply"
 
-if [[ "$#" -ne 2 ]]; then
-  echo "${0} <pr-number> <remote branch>: cherry pick <pr> onto <remote branch> and leave instructions for proposing pull request"
+if [[ "$#" -lt 2 ]]; then
+  echo "${0} <pr-number>... <remote branch>: cherry pick one or more <pr> onto <remote branch> and leave instructions for proposing pull request"
```

Comment:
totally optional: would it be easier to put the branch first, so varargs are last?